app/src/main/java/org/stepic/droid/model/code/ProgrammingLanguage.kt

```diff
@@ -106,13 +108,60 @@ enum class ProgrammingLanguage(val serverPrintableName: String) : Parcelable {
 
         }
 
-        private fun serverNameToLanguage(serverName: String): ProgrammingLanguage? {
-            return values()
-                    .find {
-                        it.serverPrintableName.equals(serverName, ignoreCase = true)
-                    } ?: return null
-        }
 
     }
 
 }
+
+private fun serverNameToLanguage(serverName: String): ProgrammingLanguage? {
+    return ProgrammingLanguage.values()
+            .find {
+                it.serverPrintableName.equals(serverName, ignoreCase = true)
+            } ?: return null
+}
+
+fun symbolsForLanguage(lang: String, context: Context): Array<String> {
+    val programmingLanguage = serverNameToLanguage(lang)
+    with(context.resources) {
+        return when (programmingLanguage) {
+            ProgrammingLanguage.PYTHON ->
+                getStringArray(R.array.autocomplete_symbols_py)
```

Comment:
Word `autocomplete` associates with keywords autocomplete that we planned to release later. Maybe it will be better to call it `frequent_symbols_py` or smth like this?